File: tests/fixtures/compare/sample.php
<?php
// Realistic PHP code with various issues
// TODO: add validation

function process($items) {
    foreach ($items as $item) {
        echo $item . "\n";
        if ($item > 10) {
            if ($item > 20) {
                echo "big: $item\n";
            }
        }
    }
}

function empty_func() {}

function complex($a, $b, $c, $d, $e) {
    return $a + $b + $c + $d + $e;
}

process([1, 2, 3, 15, 25]);

File: tests/fixtures/compare/sample.cs
// Realistic C# code with various issues
// TODO: add logging

using System;
using System.Collections.Generic;

class Sample {
    void Process(List<int> items) {
        foreach (var item in items) {
            Console.WriteLine(item);
            if (item > 10) {
                if (item > 20) {
                    Console.WriteLine("big: " + item);
                }
            }
        }
    }

    void Empty() {}

    int Complex(int a, int b, int c, int d, int e) {
        return a + b + c + d + e;
    }
}
